feat: prepare mainnet

lightscale-luke opened the pull request:
* Set Default ChainID to 255 (main net chain Id)
* Provide shell script for downloading KZG params.
* Disable creating params for KZG. Only allowing loading them.
* Set proper names for params and seed directory.

lightscale-luke committed:
feat(zkevm): set default chain id to mainnet chain id

lightscale-luke committed:
feat: disable creating params on real case

lightscale-luke committed:
refac: change default name of params directory

lightscale-luke committed:
refac: change default name of seed file

lightscale-luke committed:
feat: enable downloding KZG params with shell script

- add shell script for downloading KZG params.
- update README.md

lightscale-luke committed:
chore: remove unnecessary test_seed files

dongchangYoo commented on contrib/download_params.sh:
```diff
@@ -5,10 +5,10 @@ DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)"
 RELEASE_VERSION=params-0320
-wget https://circuit-release.s3.us-west-2.amazonaws.com/circuit-release/$RELEASE_VERSION/params20 -O ./test_params/params20
-wget https://circuit-release.s3.us-west-2.amazonaws.com/circuit-release/$RELEASE_VERSION/params26 -O ./test_params/params26
+wget https://circuit-release.s3.us-west-2.amazonaws.com/circuit-release/$RELEASE_VERSION/params20 -O ./kzg_params/params20
```
Our circuits require degrees 21 and 26.

lightscale-luke replied:
I will remove this file since it's no longer used.

fakedev9999 commented on zkevm/tests/integration.rs:
```diff
@@ -21,19 +21,19 @@ fn test_load_params() {
     log::info!("start");
     load_params(
-        "/home/ubuntu/scroll-zkevm/zkevm/test_params",
+        "/home/ubuntu/scroll-zkevm/zkevm/kzg_params",
```
Sorry, but is this the right path? scroll-zkevm?

lightscale-luke replied:
I think this path should be updated as local user's directory environment. 
I changed the last directory name for default directory name

fakedev9999 replied:
Oh it is ignored so it was fine. Maybe let's fix the path later when enabling it.

fakedev9999 reviewed (approved):
Else, looks good to me!

dongchangYoo commented:
The seed is a file, not a directory. Please see a commit title of "[refac: change default name of seed directory](https://github.com/kroma-network/kroma-prover/pull/22/commits/0f31524a0ce71b8836338ea8ce54e4e7e3cb1f27)"

lightscale-luke committed:
refac: remove unnecessary file

dongchangYoo commented:
LGTM

chokobole reviewed (approved):
LGTM

lightscale-luke committed:
feat: check CHAIN_ID env variable

- check if CHAIN_ID env is set before creating proof.
- leave prover-server starting log with chain_id.

fakedev9999 reviewed (approved):
LGTM

dongchangYoo commented:
In the "CHAIN_ID=" case, the prover works. I think it is not proper.

dongchangYoo reviewed (approved):
LGTM
